### DISCUSSION TITLE
Issue numbering

### DISCUSSION THREAD

**Discussion by mariomarinato** [community/community#43320] [New to GitHub]
My team is migrating from Trac to GitHub and all new issues will be added on GitHub after the migration takes place.

Supposing that our newest ticket on Trac is numbered 1234, is it possible to start our issues on GitHub on number 1235?

**Comment by codenay** [comment]
Yes, it is possible to start the issue numbering on GitHub at any number you choose, including 1235.

To do this, you can simply specify the number you want to start at when you create the first issue on GitHub. This can be done by adding "#1235" (or whichever number you want to start at) to the title of the issue when you create it. The issue tracker will automatically assign the next number in sequence to the next issue you create.

For example, if you create an issue with the title "Issue #1235: Fix login page bug", the next issue you create will be numbered "#1236", and so on.

Keep in mind that it may be confusing for team members if the issue numbering does not start at 1, so it might be a good idea to consider whether starting at a higher number is really necessary.

**Reply by mariomarinato** [reply]
Sounded promising but didn't work as expected.  The issues were still numbered 1 and 2.

I tried using "#number" and "Issue #number" as titles, to no avail.

**Reply by chrismit3s** [reply]
So this was never resolved?
